client/src/modules/list/components/Items/ListItem/ListItem.jsx

```diff
@@ -68,13 +71,32 @@ class ListItem extends PureComponent {
       return;
     }
 
-    this.setState(({ done }) => ({ done: !done }));
+    this.setState(({ done }) => ({ done: !done, disableToggleButton: true }));
 
     const shouldChangeAuthor = isNotSameAuthor && isOrdered;
 
-    return shouldChangeAuthor
-      ? toggle(isOrdered, _id, listId, userId, name)
-      : toggle(isOrdered, _id, listId);
+    /* setState method has to be called 100ms after toggle action is resolved
```

Comment:
I'm wondering why 100ms if it will always work properly?  Maybe it is a stupid question but what if success action will be performed longer than 100ms? Will it matter?